### DISCUSSION TITLE
Need help to make rootless containers

### DISCUSSION THREAD

**Discussion by aryanmamania** [zabbix-community/helm-zabbix#114] [Q&A] [answered]
Dear all,
I used the following command to install zabbix with helm :

aryan@aryan:~$ helm install zabbix cetic/zabbix --namespace zabbix
NAME: zabbix
LAST DEPLOYED: Thu Nov 14 20:26:32 2024
NAMESPACE: zabbix
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods --namespace zabbix -l "app=zabbix-zabbix-web" -o jsonpath="{.items[0].metadata.name}")
  export CONTAINER_PORT=$(kubectl get pod --namespace zabbix $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
  echo "Visit http://127.0.0.1:8888 to use your application"
  kubectl --namespace zabbix port-forward $POD_NAME 8888:$CONTAINER_PORT
aryan@aryan:~$ kubectl get pods -n zabbix

It worked properly and i was able to access zabbix with help of helm but when i went inside the container to install anthying i was not able to do it because it was a rootless container . I want to make a rootfull container . Anyone has any idea how to do it ?

**Comment by fibbs** [comment, marked as answer]
Sorry, you are not supposed to "install" anything inside a container. Additionally, this is a Helm Chart, not the repository for the container images. This Helm Chart is an orchestration tool to deploy and manage a Zabbix server installation in a Kubernetes Cluster, while the images we use are managed by Zabbix: https://github.com/zabbix/zabbix-docker.

I think we should close this issue.
